### ISSUE TITLE
openresty.org/package/ contains unaccessible file

### ISSUE THREAD

**Issue by ZenithalHourlyRate** [openresty/openresty.org#200]
Hi, we are [downstream mirror](https://mirrors.tuna.tsinghua.edu.cn/openresty/) of openresty. We use lftp to sync https://openresty.org/package/ to our site, the script could be found [here](https://github.com/tuna/tunasync-scripts/blob/master/lftp.sh).

Recently [all the mirror sites](https://mirrorz.org/list/openresty) using this script failed to sync from you. After investigation, it turned out that you have put unaccessible files under `package` directory, more specifically, all these excluded file below

```
lftp "https://openresty.org/package/" -e "mirror --verbose --skip-noaccess -P 5 --delete --only-newer --exclude=centos/7/2 --exclude=oracle/7/2 --exclude=rhel/7/2 --exclude=centos/7Client/2 --exclude=centos/7Server/2 --exclude=centos/8/8 --exclude=rocky/8/8 --exclude=alinux/2/2 --exclude=alinux/2.1903/2 --exclude=alinux/3/8; bye"
```

The problem is that 25 days ago (13-May-2022) you created a unaccessible file named `8` in https://openresty.org/package/centos/8/. Later a file named `2` was also created in https://openresty.org/package/centos/7/ on 25-May-2022.

Could you please remove these unnecessary files?

Also, we note that many directories are just symbol links, could you provide rsync service such that we can sync these symbol links instead of duplicating them for each directory. In that way we can save disk spaces (currently the whole openresty takes ~45G)

Also, you could use mirror site to distribute your software, like adding these mirror sites in openresty.org and redirecting traffics to them. We have also provided a redirection service for some mirror sites in https://mirrors.cernet.edu.cn/openresty so you could use it.

**Comment by xiaocang** [comment]
@ZenithalHourlyRate , thanks for the report, this should be a wrong soft link, we will remove it.

**Comment by xiaocang** [comment]
@agentzh Is it okay for us to provide `rsync` services as proposed?

**Comment by ZenithalHourlyRate** [comment]
> @agentzh Is it okay for us to provide `rsync` services as proposed?

Hi, any further update on this?

**Comment by ZenithalHourlyRate** [comment]
Close this issue as it has been fixed.



> > @agentzh Is it okay for us to provide `rsync` services as proposed?
> 
> Hi, any further update on this?

But still waiting on this. If `rsync` was provided [this new issue](https://github.com/tuna/issues/issues/1573#issuecomment-1236339317) could be avoided.
